tests/installation/welcome.pm

```diff
@@ -16,7 +16,28 @@ use warnings;
 use base "y2logsstep";
 use testapi;
 use utils 'ensure_fullscreen';
-use version_utils qw(:VERSION :SCENARIO);
+use version_utils qw(:VERSION :SCENARIO is_sle);
```

Comment:
:VERSION tag should already import `is_sle`.

Reply:
cool, I didn't know we have this constant now. good to know.